FirebaseInAppMessaging/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Unreleased
+
+- [fixed] Fixed an inaccurate doc comment in `InAppMessagingDisplay` (#5972).
```

Comment:
0.22.0 will be the next release. Move the new note into the grouping below and change date to 2020-07

Reply:
Done.